soap/src/java/com/zimbra/soap/mail/type/IdAndOperation.java

```diff
@@ -42,6 +42,13 @@
     private final String operation;
 
     /**
```

Comment:
Perhaps rename the class something like `ActionResult` now that it isn't just Id and Operation